### DISCUSSION TITLE
Which city are you from ?

### DISCUSSION THREAD

**Discussion by natsuda1da1** [NAIST-SE/AttractingContributors#1090] [2024 Attraction Contributions]
https://github.com/natsuda1da1/AttractingContributors. plz🙇

**Comment by gill876** [comment]
I'm from Falmouth, Jamaica!

Please contribute to my project!
https://github.com/gill876/AttractingContributors-Hiking

**Comment by yuanhau-cheng** [comment]
I have submit a pull request. If you have time, please also help me, Thanks a lot! (Empty pull request also ok I think)
This is my link:
https://github.com/yuanhau-cheng/Japanese-Role-Playing-Game

**Comment by Ninjia315** [comment]
I have submitted a pull request. If you have time, please also help me.
https://github.com/Ninjia315/AttractingContributors-Favorite_Japanese_songs

**Comment by Ngochuy2137** [comment]
I have just contributed to your repo. Can you contribute to my repo please https://github.com/Ngochuy2137/AttractingContributors-fruits/tree/main ?

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
